c/meterpreter/source/extensions/priv/namedpipe_efs.c

```diff
@@ -26,6 +26,10 @@ DWORD elevate_via_namedpipe_efs(Remote* remote, Packet* packet)
 	PRIV_POST_IMPERSONATION PostImpersonation;
 
 	do {
+		if (!does_pipe_exist(L"\\\\.\\pipe\\efsrpc")) {
+			BREAK_ON_ERROR("[ELEVATE] elevate_via_namedpipe_efs: \\pipe\\efsrpc is not listening.");
+		}
```

Comment:
According to https://github.com/rapid7/metasploit-framework/blob/01204106e98aacf4eb5a0537c21d5de46a240f0b/documentation/modules/post/windows/escalate/getsystem.md?plain=1#L15-L24 this should work on Windows Vista/Windows Server 2008 and later however those versions only have the `\\pipe\\lsarpc` pipe according to https://winprotocoldoc.blob.core.windows.net/productionwindowsarchives/MS-EFSR/%5BMS-EFSR%5D-150630.pdf page 83 so I'm not sure this would work unless we dropped support for Windows Vista and Windows Server 2008. Otherwise a good idea though!

Reply:
your right. I'll fix that 👍